src/NServiceBus.Persistence.Sql.TransactionalSession/SqlPersistenceOpenSessionOptions.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace NServiceBus.TransactionalSession
+{
+    using System;
+    using System.Collections.Generic;
+    using Transport;
+
+    /// <summary>
+    ///
+    /// </summary>
+    public sealed class SqlPersistenceOpenSessionOptions : OpenSessionOptions
+    {
+        /// <summary>
+        ///
+        /// </summary>
+        /// <param name="tenantIdHeaderName"></param>
+        /// <param name="tenantId"></param>
+        public SqlPersistenceOpenSessionOptions(string tenantIdHeaderName = null, string tenantId = null)
+        {
+            var headers = new Dictionary<string, string>();
+            if (tenantIdHeaderName != null)
```

Comment:
should this checks be using `IsNullOrEmpty` or `IsNullOrWhitespace`?

Reply:
I think this should be a value tuple. You always need both values

Reply:
Pushed a suggestion. Can always remove or tweak it again